CHANGELOG.md

```diff
@@ -102,7 +102,7 @@
 * Make `sorted_enum_cases` rule's comparison case-insensitive to
   avoid unexpected ordering.  
   [Oleg Kokhtenko](https://github.com/kohtenko)
-  [#5405](https://github.com/realm/SwiftLint/issues/issue_number)
+  [#5405](https://github.com/realm/SwiftLint/issues/5405)
```

Comment:
The line can actually be removed. We link to issues that were closed by the change, not to pull requests:
```suggestion
```